plugins/outputs/health/health.go

```diff
@@ -46,12 +46,17 @@ type Health struct {
 	Log      telegraf.Logger `toml:"-"`
 	checkers []Checker
 
-	wg      sync.WaitGroup
-	server  *http.Server
-	origin  string
-	network string
-	address string
-	tlsConf *tls.Config
+	UnhealthyIfMetricsAreDelayed bool            `toml:"unhealthy_if_metrics_are_delayed"`
+	MaxTimeBetweenMetrics        config.Duration `toml:"max_time_between_metrics"`
```

Comment:
With the change in the `sample.conf` you can do
```suggestion
	MaxTimeBetweenMetrics        config.Duration `toml:"max_time_between_metrics"`
```
Please move this up to before the `Log` member...

Reply:
Moved